test/migrator_test.rb

```diff
@@ -117,15 +117,15 @@
 
   describe "#shard_status" do
     it "shows nothing if everything is ok" do
-      ActiveRecord::Migrator.shard_status([1]).must_equal([{}, {}])
+      _(ActiveRecord::Migrator.shard_status([1])).must_equal([{}, {}])
```

Comment:
Can you explain this change in words, please?

Reply:
Minitest 6 introduced some deprecations: https://github.com/seattlerb/minitest/blob/96ba1dac6141599cce47ec298e7e72dc7d49812d/lib/minitest/spec.rb#L13

Reply:
FYI `require 'maxitest/global_must'` will silence this too, but converting is fine too 👍